src/hooks/useReimbursementAccountStepFormSubmit.ts

```diff
@@ -5,28 +5,29 @@ import type {OnyxFormKey} from '@src/ONYXKEYS';
 import ONYXKEYS from '@src/ONYXKEYS';
 import type {SubStepProps} from './useSubStep/types';
 
-type UseReimbursementAccountStepFormSubmitParams = Pick<SubStepProps, 'isEditing' | 'onNext'> & {
+type UseReimbursementAccountStepFormSubmitParams = Pick<SubStepProps, 'onNext'> & {
     formId?: OnyxFormKey;
     fieldIds: Array<FormOnyxKeys<typeof ONYXKEYS.FORMS.REIMBURSEMENT_ACCOUNT_FORM>>;
+    shouldSaveDraft?: boolean;
 };
 
 /**
  * Hook for handling submit method in ReimbursementAccount substeps.
  * When user is in editing mode we should save values only when user confirm that
  * @param formId - ID for particular form
- * @param isEditing - if form is in editing mode
  * @param onNext - callback
  * @param fieldIds - field IDs for particular step
+ * @param shouldSaveDraft - if we should save draft values
  */
 export default function useReimbursementAccountStepFormSubmit({
     formId = ONYXKEYS.FORMS.REIMBURSEMENT_ACCOUNT_FORM,
-    isEditing,
     onNext,
     fieldIds,
+    shouldSaveDraft = true,
```

Comment:
I appreciate your analysis and enthusiasm, but I would like to keep the behavioral changes minimal. I suggested before to rename this for the sake of clean code, not to change behavior.